codersdk/workspacebuilds.go

```diff
@@ -67,7 +67,7 @@ type WorkspaceBuild struct {
 	Job                 ProvisionerJob      `json:"job"`
 	Reason              BuildReason         `db:"reason" json:"reason"`
 	Resources           []WorkspaceResource `json:"resources"`
-	Deadline            NullTime            `json:"deadline,omitempty"`
+	Deadline            NullTime            `json:"deadline,omitempty" swaggertype:"string" format:"date-time"`
```

Comment:
Did you try to add this annotation to the actual type, `NullTime`, or is that not possible?

Reply:
I used the `.swaggo` to define a global type overwrite. Unfortunately, I can't bind permanently the format (`date-time`).

Reply:
Cool! Too bad about the type. Can we extend swaggo to support `string(date)` or something like that? Maybe we could even try to upstream it?

Reply:
The overrides file is [really simple](https://github.com/swaggo/swag/blob/f916213f43077caa4d318efb622b551cf2e45a5c/gen/gen.go#L319), so it looks like the custom format is set on a higher level. I can park it for further improvement, but wouldn't like to customize the `swag` before we cover the entire API 😅

Reply:
Oh I see what you mean, I thought this file was replacing the Go type with a swagger type but it looks like it wants Go types. I tested changing it to `time.Time` and got an unwanted result... 😂

No hurry to fix this IMO, fine to park it for now. 👍🏻